test/test.py

```diff
@@ -210,6 +210,40 @@ def test_find(session, tmpdir):
     return
 
 
+def test_case_sensitivity(session, tmpdir):
+    # Checks whether avus retain their capital letters when added
+    # Assumes the iCAT database is case-sensitive.
+    # If this is not the case, feel free to comment this testcase
+    # at the end of the script.
+
+
+    create_tmpdir(session, tmpdir)
+
+    attribute = 'kind' 
+    value1 = 'organic'
+    value2 = 'Organic'
+    
+    session.bulk.put('data/molecules/ch*', irods_path=tmpdir, verbose=True, recurse=True)
+
+    session.bulk.metadata(tmpdir+"/ch2och2.xyz", object_avu=(attribute, value1),
+                              action='add', verbose=True)
+    session.bulk.metadata(tmpdir+"/ch3cooh.xyz", object_avu=(attribute, value2),
+                              action='add', verbose=True)
+ 
+    avus = []
+    for obj in session.bulk.get(tmpdir+"/ch*", return_data_objects=True,
+                                    verbose=True):
+        value = obj.metadata.get_one(attribute).value
+        avus.append(value)
+    
+    # Comparing the two avu's
```

Comment:
Currently, the test does not seem to check that "ch2och2.xyz" has the ('kind', 'organic') AV and that "ch3cooh.xyz" has the ('kind', 'Organic') AV. It only checks that there are two such AV pairs and that their values are the same in lowercase. Just to be sure, could you modify it so that it is explicitly checked that the two molecules have the expected AV pair?